### PR TITLE
atx.intro1.morris.benjamin

### PR DESCRIPTION
**Pull request by BenjaminStanleyMorris**
This is the first pull request for Austin Coding Academy Monday/Wednesday class

### CODE REVIEW

File: 02LessonTwo/in-class-project/index.html [hunk 1]
```diff
@@ -1,10 +1,68 @@
...
-<body>
+  <head>
+    <title>Mora Law Offices</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
 
-</body>
+  <body style="background-color:powderblue;">
+    <div align="center">
+      <nav>
```
Comment (e-zambrano) [inline 02LessonTwo/in-class-project/index.html:10]: nice use of a semantic element

File: 02LessonTwo/in-class-project/index.html [hunk 2]
```diff
@@ -1,10 +1,68 @@
...
-  <link rel="stylesheet" href="css/style.css">
-</head>
-<body>
+  <head>
+    <title>Mora Law Offices</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
 
-</body>
+  <body style="background-color:powderblue;">
```
Comment (e-zambrano) [inline 02LessonTwo/in-class-project/index.html:8]: we are staying away from in-line style from now on

**Comment by e-zambrano** [comment]
@BenjaminStanleyMorris - high five

**Comment by e-zambrano** [comment]
@BenjaminStanleyMorris - awesome job adding JavaScript. Just avoid committing files from different homeworks together.

**Comment by e-zambrano** [comment]
@BenjaminStanleyMorris - it's very hard to find your code for each exercise or homework, everything seems to be labeled to generic or mentioning some merge to a branch.
You are having issues with floats, but didn't see you asking question or requesting to review that topic. Make sure you watch the lesson and try to fix those project that have low grade

**Comment by BenjaminStanleyMorris** [comment]
Hey Erik,

What is a better label for the exercises? I also commit quite often so I
can work on my desktop. Floating is still a bit confusing, but I will work
through it and make sure I fix those lessons with a lower grade. I have
bootstrapped my index page to get some more experience working with that. I
think I told you but I am going to tech crunch for work that week for work
and I will miss class Monday and Tuesday.

Best,
Benjamin

File: 10LessonTen/Checkpoint2/index.html [hunk 3]
```diff
@@ -5,10 +5,20 @@
     <title>Checkpoint 2</title>
     <link rel="stylesheet" href="css/css-reset.css">
     <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Cabin" rel="stylesheet">
+    <script>
+    function alertFunctionRollTide() {
```
Comment (e-zambrano) [inline 10LessonTen/Checkpoint2/index.html:10]: Look at you using some javascript

**Comment by e-zambrano** [comment]
@BenjaminStanleyMorris - Good work here. Just remember to **beautify** your code before committing your code.

File: 13LessonThirteen/homework/index.html [hunk 4]
```diff
@@ -1,10 +1,31 @@
...
 <body>
+  <container>
+  <div class="jumbotron">
+    <h2>We're creatives</h2>
+  </div>
+</container>
+<container class="middleSection">
+<row class="topMiddleRow">
+<div class="col-lg-4 rowImg">
+  <img src="http://bone.themanoid.com/images/thumbs/07.png" height="250px" alt="Beautiful Mountain Scape" />
```
Comment (zacacollier) [inline 13LessonThirteen/homework/index.html:20]: Best practice is to avoid inline styling

**Pull request closed without merging**